test/blackbox-tests/test-cases/select-in-test.t

```diff
@@ -0,0 +1,32 @@
+We test the (select) field of the (libraries) field of the test stanza.
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.11)
+  > EOF
+
+  $ cat > dune <<EOF
+  > (test
+  >  (name test)
+  >  (modules test)
```

Comment:
Do you need the `modules` line to reproduce the bug?

Reply:
No, that was from when the other stanzas were there. I'll remove it.